### DISCUSSION TITLE
Can you share frontend from and listing screens?

### DISCUSSION THREAD

**Discussion by umefarooq** [TepuiLABS/simple-crm#8] [Ideas]
Hi, its look really nice package. Can you share the front-end of the crm also how listing looks like and forms for lead, project and services. It will be really helpful. This package will be nice addition to projects with simple CRM.

I will be really thankful for your support.

**Comment by abr4xas** [comment]
I will try to do that but any PR are welcome.

**Comment by umefarooq** [comment]
Just share the screenshot of forms will send you PR

**Reply by abr4xas** [reply]
Hey @umefarooq sorry for the delay, I think this can help:

![table view | https://laravelviews.com/ ](https://github.com/TepuiLABS/simple-crm/assets/405484/d57e6f87-e70f-4219-acb1-c35afc2999eb)

**Comment by umefarooq** [comment]
Thank you for reply, I am looking for form I have check other laravel crm like daybyday crm form got some ideas for using this package.
